Applications/DataExplorer/VtkAct/VtkCustomInteractorStyle.h

```diff
@@ -72,13 +72,13 @@ public slots:
 	virtual ~VtkCustomInteractorStyle();
 
 	/// @brief The vtk object to pick.
-	vtkDataObject* Data;
+	vtkDataObject* _data;
 
 	/// @brief The mapper for highlighting the selected cell.
-	vtkDataSetMapper* selectedMapper;
+	vtkDataSetMapper* _selectedMapper;
 
 	/// @brief The actor for highlighting the selected cell.
-	vtkActor* selectedActor;
+	vtkActor* _selectedActor;
```

Comment:
Maybe it is possible to use `vtkSmartPointer` or `std::unique_ptr` for `_data` (line 75), `_selectedMapper` (line 78) or `_selectedActor`?